Disable tagfiles, this triggers a segfault in doxygen-1.8.6 while processing
the gepetto-viewer-corba.doxytag

This does not change the documentation anyway: the generated "related pages" is
empty even with tagfiles on.

--- doc/Doxyfile.extra.in~	2015-03-04 16:10:14.000000000 +0100
+++ doc/Doxyfile.extra.in	2018-03-23 17:36:58.822322330 +0100
@@ -1,5 +1,3 @@
-TAGFILES		 = @HPP_CORBASERVER_DOXYGENDOCDIR@/hpp-corbaserver.doxytag=@HPP_CORBASERVER_DOXYGENDOCDIR@ \
-			   @GEPETTO_VIEWER_CORBA_DOXYGENDOCDIR@/gepetto-viewer-corba.doxytag=@GEPETTO_VIEWER_CORBA_DOXYGENDOCDIR@
 
 INPUT			 = @CMAKE_SOURCE_DIR@/src/hpp/gepetto \
 			   @CMAKE_SOURCE_DIR@/doc
